### DISCUSSION TITLE
Laure Prouvost + LAS Foundation "We Felt a Star Dying" |  Kraftwerk Berlin (February 21–May 4, 2025)

### DISCUSSION THREAD

**Discussion by studiofomalhaut** [Quantumland-art/community#83] [Art]
Turner Prize winner Laure Prouvost has been commissioned by LAS Foundation to occupy the large hall of Kraftwerk Berlin with the installation "_We Felt a Star Dying"_ — a plant-like installation made in collaboration with the philosopher Tobias Rees and the Google Quantum AI team. It goes on show on 20 February, which is still a few months away, but the word is out! 

_"(...)the result is a multi-sensory and immersive environment, including video developed with a quantum computer, as well as sound, scent and sculptural elements tuned to the highly sensitive and unpredictable characteristics of quantum computers. The project merges Prouvost’s playful and intuitive approach with the counterintuitive logic of quantum physics and represents a new line of enquiry in the artist’s oeuvre. Presented over 3,500 square metres (37,600 square feet) of former power station Kraftwerk Berlin, the commission offers a dynamic reflection on technological development today."_

<img width="1386" alt="Screenshot 2024-12-12 at 11 21 03" src="https://github.com/user-attachments/assets/dd903cde-a1d2-4910-82f5-12715cf926c3" />

To read more see [LAS Foundation](https://www.las-art.foundation/programme/we-felt-a-star-dying) and Lisson Gallery's [press release](https://www.lissongallery.com/news/las-art-foundation-presents-laure-prouvost-we-felt-a-star-dying-at-kraftwerk-berlin)

**Comment by studiofomalhaut** [comment]
On this topic, some articles on [Frieze Magazine](https://www.frieze.com/article/will-quantum-computing-change-art) and [Art Review](https://artreview.com/how-art-went-quantum-laure-prouvost-las-foundation-cern-ai-opinion-gunseli-yalcinkaya/) have just been published

**Comment by omarcostahamido** [comment]
Just in: 
winner of S+T+ARTS `Grand Prize: Innovative Collaboration`🏆

https://ars.electronica.art/starts-prize/en/jury/jury2025/

**Reply by studiofomalhaut** [reply]
That' good news :) and well deserved
